pages/roadmap.js

```diff
@@ -0,0 +1,112 @@
+import Image from "next/image";
+import {
+  FaCog,
+  FaCogs,
+  FaDraftingCompass,
+  FaPalette,
+  FaSplotch,
+  FaCodeBranch,
+  FaShieldAlt,
+  FaCloudUploadAlt,
+} from "react-icons/fa";
+
+import RoadMapCard from "../components/RoadMapCard.js";
+import { AiFillHtml5 } from "react-icons/ai";
+const RoadMap = () => {
+  return (
+    <main>
+      <div className=" w-screen bg-light flex">
+        <span className="w-1/2 p-28 ">
+          <h1 className="font-bold text-dark text-5xl">Delveloper Roadmaps</h1>
+          <p className="font-medium text-dark">
+            Follow these roadmaps and become a hero in any path you choose!
+          </p>
+        </span>
+        <span className="w-1/2   ">
+          <Image
+            src="/On the way-amico 1.png"
+            alt="roadmap"
+            width={500}
+            height={500}
+          />
+        </span>
+      </div>
+      <div className="p-24 space-y-14 ">
+        <h1 className="text-dark font-semi text-4xl">Web Development</h1>
```

Comment:
@ayasalam0 i think you meant to write semibold this typo has occurred in several places, please fix